### ISSUE TITLE
android only: ui-material-tabs > 6.2.20 breaks nativescript-ui-calendar 

### ISSUE THREAD

**Issue by cjohn001** [nativescript-community/ui-material-components#372]
I have a nativescrip-ui-calender in a page displayed within the ui-material-tabs. On Android, the styles of the calendar are broken when it is loaded. In addition to the basic styles I am configuring some parameters of the calendar in the loaded event

		if (global.isAndroid) {
			// disable gestures to switch between view modes
			const gestureManager = this._radCalendar.nativeView.getGestureManager();
			gestureManager.setPinchOpenToChangeDisplayMode(false);
			gestureManager.setPinchCloseToChangeDisplayMode(false);
			gestureManager.setSwipeUpToChangeDisplayMode(false);
			gestureManager.setSwipeDownToChangeDisplayMode(false);
			gestureManager.setTapToChangeDisplayMode(false);
			gestureManager.setDoubleTapToChangeDisplayMode(false);
			this._radCalendar.android.setDayNamesHeight(Utils.layout.toDevicePixels(18));
			this._radCalendar.android.setTitleHeight(30 * Screen.mainScreen.scale);
			this._radCalendar.android.setInOriginalSizeForAllModes(true);
		}
The following lines result in the calendar disappearing at all
	this._radCalendar.android.setDayNamesHeight(Utils.layout.toDevicePixels(18));
	this._radCalendar.android.setTitleHeight(30 * Screen.mainScreen.scale);
	this._radCalendar.android.setInOriginalSizeForAllModes(true);
Seems like the view layout is not correctly updated when it is in a page displayed in the tabs component

### Which platform(s) does your issue occur on?
- Android API 28 tried on Emulator und device

### Please, provide the following version numbers that your issue occurs with:

- CLI: 8.2.2
- Cross-platform modules: "@nativescript/core": "8.2.1",
- Runtime(s): "@nativescript/android": "8.2.2"
- Plugin(s):  "nativescript-ui-calendar":"8.0.2"

**Comment by farfromrefug** [comment]
@cjohn001 this issue should be reported in the calendar plugin. i cant support any of the ui plugin as they are closed source.

**Comment by cjohn001** [comment]
@farfromrefug : I am rather sure that this is a problem related to the rewrite of the android version of MDTabs. On MDTabs till version 6.2.20 this works as expected and I am using the calendar plugin in at least 5 different pages in the same app ,exactly same source as on the MDTabs page, without issues

**Comment by farfromrefug** [comment]
@cjohn001 yes you are right it is related to that change. we now use the normal fragment technique .
now if you say.that the calendar works without calling those lines then i can it should be reported in the calendar repo. again as they are closed source i have no idea of what s happening behind and i can neither say the issue is here nor fix it
